tests/test_castanet.py

```diff
@@ -0,0 +1,29 @@
+"""A file for testing the functionality of counter.py."""
+
+from castanet import generate_trees as generator
+from castanet import counter
+
+def test_import_dictionary():
+    """Test that the dictionary for imports is being generated correctly"""
```

Comment:
There is a missing period on the docstring here, but every other docstring looks good.